client/src/components/Question.js

```diff
@@ -1,28 +1,44 @@
 import React from 'react';
-import { Form, Jumbotron } from 'react-bootstrap';
+import { Form, Card } from 'react-bootstrap';
 
 const Question = ({ question, handleAnswerSelect, selectedAnswer = [] }) => {
+  const addClassName = question => {
+    if (question.qType === 'checkbox') {
+      return 'styled-input-square';
+    } else {
+      return 'styled-input-rounded';
+    }
+  };
+
   return (
     <fieldset>
-      <Jumbotron>
-        <h2>{question.text}</h2>
-        <Form.Group key={question._id}>
-          {question.answers.map(answer => {
-            return (
-              <Form.Check
-                key={answer._id}
-                type={question.qType}
-                id={`${question._id}.${answer._id}`}
-                name={question._id}
-                value={answer._id}
-                onChange={e => handleAnswerSelect(e, question.qType)}
-                label={`${answer.text}`}
-                defaultChecked={selectedAnswer.includes(answer._id)}
-              />
-            );
-          })}
-        </Form.Group>
-      </Jumbotron>
+      <Card className="mb-3 rounded">
+        <Card.Header className="bg-secondary text-white" as="h5">
```

Comment:
```suggestion
        <Card.Header className="bg-secondary text-white" as="h2">
```
Headings should be sequential. There is an `h1` earlier in the page that displays `HTML quiz`, but no other headings so this heading should then be an `h2`. If the size is too big you can use CSS to adjust it, for example by adding the class `.h2` (see https://getbootstrap.com/docs/4.4/content/typography/#headings)